validations.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# require_relative("event")
+require('time')
+require('date')
+require 'io/console'
+# Validations for validations and helper methods
+module Validations
+  
+
+  def validate_date(date)
+    return nil if (date =~ %r(^\d{2}/\d{2}/\d{4})).nil? && (date =~ %r(^\d{2}/\d{4})).nil?
+
+    Date.parse(date)
+  rescue StandardError
+    nil
+  end
+
+  def validate_time(time)
+    rettime = get_time(time)
+    return Time.parse(rettime) unless rettime.nil?
+
+    nil
+  end
+
+  def validate_integer(num)
+    num.to_i
+  rescue StandardError
```

Comment:
to_i would not throw exception as any invalid integer string would parse to integer 0, so no need to validate integer

Reply:
use num.to_i directly wherever this method is used